README.md

```diff
@@ -210,106 +185,91 @@ Further, the key manager interface **must** be passed as an argument to _all_ pu
-| Validation          | ❌         | ❌     | ❌   | ❌    |
-
-### [Bitstring Status List](https://w3c.github.io/vc-bitstring-status-list/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation using `vc-jwt`           | ❌         | ❌     | ❌   | ❌   |
-| Validation using `vc-jwt`         | ❌         | ❌     | ❌   | ❌   |
-
-### [VC JSON Schema](https://www.w3.org/TR/vc-json-schema/)
-
-| Feature                           | Typescript | Kotlin | Rust | Swift |
-| --------------------------------- | ---------- | ------ | ---- | ----- |
-| Creation `JsonSchema`             | ❌         | ❌     | ❌   | ❌   |
-| Creation `JsonSchemaCredential`   | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchema`           | ❌         | ❌     | ❌   | ❌   |
-| Validation `JsonSchemaCredential` | ❌         | ❌     | ❌   | ❌   |
-
-### [Presentation Exchange V2](https://identity.foundation/presentation-exchange/spec/v2.0.0/)
-
-| Feature                | Typescript | Kotlin | Rust | Swift |
-| ---------------------- | ---------- | ------ | ---- | ----- |
-| Concrete Type          | ✅         | ✅     | ❌   | ❌    |
-| Validation             | ✅         | ✅     | ❌   | ❌    |
-| Credential Evaluation  | ✅         | ✅     | ❌   | ❌    |
-| [Predicates](https://identity.foundation/presentation-exchange/spec/v2.0.0/#predicate-feature)                           | ✅         | ✅     | ❌   | ❌    |
-| [Relational Constraints](https://identity.foundation/presentation-exchange/spec/v2.0.0/#relational-constraint-feature)   | ✅         | ❌     | ❌   | ❌    |
-| [Credential Status](https://identity.foundation/presentation-exchange/spec/v2.0.0/#credential-status-constraint-feature) | ❌         | ❌     | ❌   | ❌    |
+| Feature                  |
+| ------------------------ |
+| Data model               | 
+| Signing and verification | 
+
+#### [Status List 2021](https://www.w3.org/community/reports/credentials/CG-FINAL-vc-status-list-2021-20230102/)
+
+For usage with the [W3C Verifiable Credential Data Model 1.1](https://www.w3.org/TR/vc-data-model).
+
+| Feature                                | 
+| -------------------------------------- | 
+| Data model                             |
+| Status checking                        |
+| Status setting                         |
+| Signing and verification with `vc-jwt` |
+
+#### [Bitstring Status List](https://www.w3.org/TR/vc-bitstring-status-list/)
+
+For usage with the [W3C Verifiable Credential Data Model 2.0](https://www.w3.org/TR/vc-data-model-2.0/).
+
+| Feature                                      | 
+| -------------------------------------------- | 
+| Data model                                   |
+| Status checking                              |
+| Status setting                               |
+| Signing and verification with `vc-jose-cose` |
+
+
+#### [VC JSON Schema](https://www.w3.org/TR/vc-json-schema/)
```

Comment:
from my expirence trying to validate a vc / pex object based on schema, it seemed like json schema was not robust enough to handle all the edge cases. 

For example, Robust stuff like checking for unique input descriptors, I found that json schema could not save me. I had to make a validator -  https://github.com/TBD54566975/web5-kt/blob/main/credentials/src/main/kotlin/web5/sdk/credentials/model/Validator.kt#L44

Reply:
correct, it's just a first line of defense. business logic needs real validation code